### PR TITLE
Improvement/modify env variables and throw errors when missing in dev. 

### PR DESCRIPTION
**Pull request by john-raymon**

### CODE REVIEW

**Comment by ScopeyNZ** [comment]
We don't usually commit `.env` files into VCS. Can you remove that?

**Comment by ScopeyNZ** [comment]
We don't usually commit `.env` files into VCS. Can you remove that?

File: src/lib/commerce.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import Commerce from '@chec/commerce.js';
 
+const commercejsPublicKey = process.env.REACT_APP_COMMERCEJS_PUBLIC_KEY;
+const isDevMode = process.env.NODE_ENV === 'development';
+if (isDevMode && !commercejsPublicKey) {
+  throw Error('You must provide your public API key as an environment variable named REACT_APP_COMMERCEJS_PUBLIC_KEY for commerce.js to work. Your public key is available within your Chec dashboard.')
```
Comment (robbieaverill) [inline src/lib/commerce.js]:
```suggestion
  throw Error('You must provide your public API key as an environment variable named REACT_APP_COMMERCEJS_PUBLIC_KEY for Commerce.js to work. Your public key is available within your Chec dashboard, or can be obtained with the Chec CLI via "whoami".')
```

File: .env.example [hunk 2]
```diff
@@ -1,2 +1,6 @@
 REACT_APP_COMMERCEJS_PUBLIC_KEY=
+/* CHEC_SECRET_KEY used with chec/seeder to access your Chec dashboard, seeding it with sample products  */
```
Comment (robbieaverill) [inline .env.example]:
```suggestion
# CHEC_SECRET_KEY used with chec/seeder to access your Chec dashboard, seeding it with sample products 
```

File: .env.example [hunk 3]
```diff
@@ -1,2 +1,6 @@
...
+/* Recommended to remove once done using with chec/seeder. */
```
Comment (robbieaverill) [inline .env.example]:
```suggestion
# Recommended to remove once done using with chec/seeder
```

**Review by robbieaverill — APPROVED** [review]

**Pull request merged**